docs/src/main/asciidoc/amqp-reference.adoc

```diff
@@ -478,6 +478,19 @@ public AmqpClientOptions getNamedOptions() {
 }
 ----
 
+== TLS Configuration
+
+AMQP 1.0 Messaging extension integrates with the xref:./tls-registry-reference.adoc[Quarkus TLS registry] to configure the Vert.x AMQP client.
+
+To configure the TLS for an AMQP 1.0 channel, you need to provide a named TLS configuration (or `<default>`, for the default configuration) in the `application.properties`:
```

Comment:
Following the ADR 0004 (https://github.com/quarkusio/quarkus/blob/main/adr/0004-using-the-tls-registry-for-clients.adoc) - it is recommended to use a named. So, I would remove the reference to `<default>`.